frontend/public/css/styles.scss

```diff
@@ -690,6 +690,7 @@ html.js {
 }
 
 #right0 {
+    margin-top: 13px;
```

Comment:
By re-positioning the save and revert buttons this value will maybe change. If so, think about how to express it in relative, instead of absolute, terms.